### DISCUSSION TITLE
GEVI Manual Identification

### DISCUSSION THREAD

**Discussion by ilouotros** [CodyBerenson/PGMA-Modernized#281] [Ideas]
As an idea for easier matching, it could be possible to manually specify the film in the criterias using it's id.
For example: 
A file is named (Kristen Bjorn Video) - Orgies 1 (2012).
Using automatic match it doesn't match the film so I go to GEVI and search it manually finding it's https://gayeroticvideoindex.com/video/45466.

In the criteria fields if I write GEVI-45466, the scanner doesn't need to search in all GEVI, it assigns the film which is in that URL.
![Captura](https://github.com/CodyBerenson/PGMA-Modernized/assets/63206647/2a6a0b71-39ec-4ef8-8cb7-172fb4317847).
So it can run with movies as well with scenes, it could be even more complete if you write something like:
GEVI-MOVIE-45466 --> It is a movie
GEVI-SCENE-30249 --> It is a scene

**Comment by JPH71** [comment]
Now that is an idea worth looking at... though what we did for the scenes
was a special case...
We could expand the gevi scenes matcher....
But on the other hand GEVI works well...
By doing this that means you will have to include this code in the filename
and the other agents would not find it...
If GEVI for one reason or another went off line you would be literally at
the bottom of a pit latrine....
